app/document/content/application-details.js

```diff
@@ -20,20 +20,24 @@ const applicationDetails = (data) => {
   return {
     stack: [
       { text: 'You have applied for funding for a review.\n\n' },
-      { text: 'By applying for this funding you’ve entered into an agreement with the Rural Payments Agency (RPA). The agreement will be governed by the agreed terms and conditions.\n\n' },
+      { text: 'By applying for this funding you have entered into an agreement with the Rural Payments Agency (RPA). The agreement will be governed by the agreed ' },
```

Comment:
why space at the end?

Reply:
So the text doesn't run into the link text which is part of the same sentence.

Reply:
The actual text content has been split into two lines of code so a space is required when they are joined up. They have been split as the text in the 2nd line of code is a link so requires special formatting.